doc/whats_new.rst

```diff
@@ -31,6 +31,14 @@ Changelog
 New features
 ............
 
+   - :class:`model_selection.GridSearchCV`,
+     :class:`model_selection.RandomizedSearchCV` now support multimetric
+     evaluation. Refer the :ref:`multimetric_grid_search` section of userguide
+     for more information. :issue:`7388` by `Raghav RV`_
+
+   - :func:`model_selection.cross_val_score` now support multimetric
```

Comment:
Maybe make this one item? Or do you want to have the two different links to the user guide? You could still keep those...